Detectors/FIT/FT0/workflow/include/FT0Workflow/RawReaderFT0.h

```diff
@@ -121,17 +129,24 @@ class RawReaderFT0ext : public RawReaderFT0BaseExt
     outputSpec.emplace_back(o2::header::gDataOriginFT0, "DIGITSBC", 0, o2::framework::Lifetime::Timeframe);
     outputSpec.emplace_back(o2::header::gDataOriginFT0, "DIGITSCH", 0, o2::framework::Lifetime::Timeframe);
     outputSpec.emplace_back(o2::header::gDataOriginFT0, "DIGITSTRGEXT", 0, o2::framework::Lifetime::Timeframe);
+    if constexpr (sUseTrgInput) {
+      outputSpec.emplace_back(o2::header::gDataOriginFT0, "TRIGGERINPUT", 0, o2::framework::Lifetime::Timeframe);
+    }
   }
   void makeSnapshot(o2::framework::ProcessingContext& pc)
   {
     pc.outputs().snapshot(o2::framework::Output{o2::header::gDataOriginFT0, "DIGITSBC", 0, o2::framework::Lifetime::Timeframe}, mVecDigitsExt);
     pc.outputs().snapshot(o2::framework::Output{o2::header::gDataOriginFT0, "DIGITSCH", 0, o2::framework::Lifetime::Timeframe}, mVecChannelData);
     pc.outputs().snapshot(o2::framework::Output{o2::header::gDataOriginFT0, "DIGITSTRGEXT", 0, o2::framework::Lifetime::Timeframe}, mVecTrgExt);
+    if constexpr (sUseTrgInput) {
+      pc.outputs().snapshot(o2::framework::Output{o2::header::gDataOriginFT0, "TRIGGERINPUT", 0, o2::framework::Lifetime::Timeframe}, mVecTriggerInput);
+    }
```

Comment:
@afurs just to be sure: when do we need a trigger input created (?) from the raw data? 
Note that in the standard simulation it is needed to generate the CTP "digits" (trigger decision) which eventually will become a "raw data" on its own.

Reply:
Unfortunately, I don't know. I decided to repeat it(TriggerInput creation) also for extended mode(RawReaderFT0ext), just in case.